platform/core/src/DICOMSR/SCOORD3D/utils/getRenderableData.js

```diff
@@ -1,9 +1,24 @@
 import csMath from 'cornerstone-math';
 import SCOORD_TYPES from '../constants/scoordTypes';
 
-const getRenderableData = (GraphicType, GraphicData, ValueType) => {
+const getRenderableData = (
+  GraphicType,
+  GraphicData,
+  ValueType,
+  imageMetadata
+) => {
   let renderableData;
 
+  // Question for Steve and Andrey, with study 1.3.6.1.4.1.14519.5.2.1.6279.6001.100063870746088919758706456900
+  // (the one with ~1000 polygons), looks like the SCOORD3D coordinates are given respect to the
+  // center of the image, however in the dicom refers that the coordinates are given respect to
```

Comment:
I saw these comments in other parts of the code and I think moving forward we can stick with this style:

```
/** 
 * Some random text...
 * Some random text...
 */
```
 
 It looks better / follows the same pattern as jsDocs.

Reply:
fixed all